plugins/forc-index/src/commands/deploy.rs

```diff
@@ -30,11 +30,12 @@ pub struct Command {
     #[clap(long, default_value = defaults::INDEXER_TARGET, help = "Target at which to compile.")]
     pub target: String,
 
-    /// Build optimized artifacts with the release profile.
+    /// Build optimized artifacts with the release profile. Set to true by default.
     #[clap(
         short,
         long,
-        help = "Build optimized artifacts with the release profile."
+        help = "Build optimized artifacts with the release profile.",
+        default_value = "true"
```

Comment:
Could we keep the pattern of using default values?

```rust
// forc-index/src/utils/defaults.rs

// ...

pub const BUILD_RELEASE_PROFILE: bool = true; // or whatever you wanna call it
```
And then 

```rust
  #[clap(
         short,
         long,
         help = "Build optimized artifacts with the release profile.",
         default_value_t = defaults::BUILD_RELEASE_PROFILE,

```

Reply:
Updated in the new commit